.github/workflows/rebase-ci-bootstrap.yaml

```diff
@@ -0,0 +1,35 @@
+name: Rebase ci/bootstrap on main
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  rebase:
+    runs-on: ubuntu-22.04
+    steps:
+
+    - uses: actions/checkout@v4
+      with:
+        ref: ci/bootstrap
```

Comment:
This could be run as a matrix job instead to avoid duplication, if one would like to.
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
I did consider this ... but then had to consciously resist the urgh to optimise :) Wdty about going with the simpler approach for now and then revisiting if/when we expand this out to two cases?